fe/fe-core/src/main/java/org/apache/doris/load/ExportJob.java

```diff
@@ -266,17 +278,101 @@ private void generateQueryStmt() {
+            selectStmtList.add(selectStmt);
+        }
+        stmtExecutorList = Arrays.asList(new StmtExecutor[selectStmtList.size()]);
+        if (LOG.isDebugEnabled()) {
+            for (int i = 0; i < selectStmtList.size(); i++) {
+                LOG.debug("Outfile clause {} is: {}", i, selectStmtList.get(i).toSql());
+            }
+        }
+    }
+
+    private ArrayList<ArrayList<TableRef>> splitTablets(ExportStmt stmt) throws UserException {
+        // get tablets
+        Database db = Env.getCurrentEnv().getInternalCatalog().getDbOrAnalysisException(stmt.getTblName().getDb());
+        OlapTable table = db.getOlapTableOrAnalysisException(stmt.getTblName().getTbl());
+        List<Long> tabletIdList = Lists.newArrayList();
+        table.readLock();
+        try {
+            Collection<Partition> partitions = new ArrayList<Partition>();
+            // get partitions
+            // user specifies partitions, already checked in ExportStmt
+            if (this.partitionNames != null) {
+                if (partitionNames.size() > Config.maximum_number_of_export_partitions) {
+                    throw new UserException("The partitions number of this export job is larger than the maximum number"
+                            + " of partitions allowed by a export job");
+                }
+                for (String partName : this.partitionNames) {
+                    partitions.add(table.getPartition(partName));
+                }
+            } else {
+                if (table.getPartitions().size() > Config.maximum_number_of_export_partitions) {
+                    throw new UserException("The partitions number of this export job is larger than the maximum number"
+                            + " of partitions allowed by a export job");
+                }
+                partitions = table.getPartitions();
+            }
+
+            // get tablets
+            for (Partition partition : partitions) {
+                partition.getVisibleVersion();
+                partitionToVersion.put(partition.getName(), partition.getVisibleVersion());
+                for (MaterializedIndex index : partition.getMaterializedIndices(IndexExtState.VISIBLE)) {
+                    tabletIdList.addAll(index.getTabletIdsInOrder());
+                }
+            }
+        } finally {
+            table.readUnlock();
+        }
+
+        Integer tabletsAllNum = tabletIdList.size();
+        Integer tabletsNumPerQuery = tabletsAllNum / this.parallelNum;
+        Integer tabletsNumPerQueryRemainder = tabletsAllNum - tabletsNumPerQuery * this.parallelNum;
+
+        Integer start = 0;
+
+        ArrayList<ArrayList<TableRef>> tableRefListPerQuery = Lists.newArrayList();
+
+        int outfileNum = this.parallelNum;
+        if (tabletsAllNum < this.parallelNum) {
+            outfileNum = tabletsAllNum;
+            LOG.warn("The number of tablets is smaller than parallelism, set parallelism to tablets num.");
```

Comment:
This log does not contains any useful info, eg, job id, number of tablet, parallelism, etc.